e2e-tests/support/tasks.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
        * Deletes all the rows from the utilisation report and azure file info tables
        */
       async removeAllUtilisationReportsFromDb() {
+        await SqlDbDataSource.manager.getRepository(UtilisationDataEntity).delete({});
```

Comment:
This is required as we don't have cascade deletes for `UtilisationDataEntity`, so we need to delete those first